setup.cfg

```diff
@@ -8,7 +8,7 @@ author_email = is-devops-team@canonical.com
 [options]
 packages = find:
 include_package_data = true
-python_requires = ~=3.8
+python_requires = ~=3.12.2
```

Comment:
3.12 should suffice, as python follows semantic versioning

Reply:
got it, thanks!

Reply:
I tried putting 3.12 but poetry compained since on indico 3.3.1 it is `~=3.12.2`, so I'll leave it as it is for now